WORKSPACE

```diff
@@ -131,6 +131,24 @@ new_go_repository(
 )
 
 new_go_repository(
```

Comment:
Do we use these new repos anywhere? I couldn't spot them in the BUILD files/imports.